tox.ini

```diff
@@ -6,11 +6,13 @@
 [tox]
 envlist = py{27,34,35,36,py}-pytest30-supported-xdist,
           py{27,35}-pytest30-unsupported-xdist,
-          py{27,34,py}-pytest31-supported-xdist
-          py{27,34,py}-pytest34-supported-xdist
+          py{27,34,py}-pytest31-supported-xdist,
+          py{27,34,py}-pytest34-supported-xdist,
+          py37-pytest{39,310}-supported-xdist,
           qa
 
 [testenv]
+usedevelop = True
```

Comment:
I guess this was done to get the coverage right?
While it is faster, I think it is better to not use this on CI.

Reply:
Yes it is so that we use the right file for the coverage.